style.css

```diff
@@ -250,3 +250,70 @@ nav {
   color: #253858;
   font-size: 15px;
 }
+
+.form-section {
+  background-color: white;
+}
+
+.form-section p {
+  margin: 25px 0 30px 0;
+  color: white;
+}
+
+.form-section h2 {
+  font-size: 40px;
+}
+
+form {
+  display: flex;
+  background: #6070ff;
+  flex-direction: column;
+  justify-content: space-between;
+  border-top-left-radius: 100px;
+  padding: 150px 5% 50px 5%;
+  color: white;
+}
+
+input,
+textarea {
+  margin-bottom: 20px;
+  border-radius: 5px;
+  padding: 10px;
+  border: none;
+}
+
+.form-section button {
+  width: 100px;
```

Comment:
- [ ] It will be better if you give button `witdth: 120px;` to match it like the design.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/192518744-ed2f7bc3-a332-4c28-9a4d-0e6cf9248b2a.png) | ![image](https://user-images.githubusercontent.com/53402600/192518541-1fc5e070-e9a1-4825-8941-e0c3474b27fd.png) |